deepchem/feat/atomic_coordinates.py

```diff
@@ -10,80 +10,170 @@
 __license__ = "LGPL v2.1+"
 
 import numpy as np
-import mdtraj
+from rdkit import Chem
 from deepchem.feat import Featurizer
 from deepchem.feat import ComplexFeaturizer
-from deepchem.utils import rdkit_util
+from deepchem.utils import pad_array, rdkit_util
 
 
-class AtomicCoordinates(Featurizer):
```

Comment:
The atomic-convs code seems to use the old neighbor-listing implementation I did in pure python instead of using `mdtraj`'s optimized version that @peastman added in a few PRs ago. Might be worth just making the change to use the `mdtraj` implementation which is much faster rather than my custom implementation.